src/components/typography/text/text.js

```diff
@@ -17,17 +22,15 @@ const Headline = props => {
   const HeadlineElement = props.elementType;
   return (
     <HeadlineElement
-      className={classnames({
-        [styles.headline]: true,
-        [styles.truncate]: props.truncate,
-      })}
+      css={headlineStyles(props)}
```

Comment:
Given we likely will spread this "pattern":

1. Maybe `getHeadline{Styles,Css}?
2. Should we pass only what the function needs from props or always everything to keep things simple?

Reply:
yeah it makes sense to make these decisions now when it's easy to refactor 👍